content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +13,139 @@ further_reading:
+Steps are a series of actions that you can record for a browser test and edit or build on. To define the steps you want your browser test to execute, either directly record them with the Datadog test recorder extension or add them manually. Every step includes a set of configurable [advanced options][1].
 
-**Note**: The default timeout for each step is approximately 60 seconds. You can override this default timeout through the dedicated [advanced option][2].
+The default timeout for each step is approximately 60 seconds. You can override this default timeout through the dedicated [timeout option][2]. 
+
+## Manage step order
+
+Instead of manually reordering new steps by dragging and dropping individual steps, you can set a cursor on a test step at a particular stage in your recording and insert additional steps. 
+
+1. Hover over a recorded test step and click the **Set Cursor** icon. A blue line appears above your test step. 
+2. Record additional [test steps](#automatically-recorded-steps) or add [steps manually](#manually-added-steps).
+3. When you complete adding additional steps above your tests step, click **Clear Cursor** to exit.
+
+{{< img src="synthetics/browser_tests/recording_cursor_step.mp4" alt="Set the cursor on a test step to add additional steps before this step" video="true" width="100%" >}}
 
 ## Automatically recorded steps
 
-The following steps are automatically recorded with the [Datadog browser test recorder extension][3]:
+Once you click **Start Recording**, the [Datadog browser test recorder extension][3] automatically detects and records steps on your website.
 
 ### Click
 
-[Datadog browser test recorder extension][3] automatically records clicks on your page. Specify the type of click you want your browser test to perform at execution time:
+Interacting with elements on your page records a step on the left corner. 
 
-{{< img src="synthetics/browser_tests/browser_test_click_step.mp4" alt="Browser Test Click Step" video="true" width="60%">}}
+{{< img src="synthetics/browser_tests/click_step.mp4" alt="Click type dropdown menu in the Click step type" video="true" width="60%" >}}
 
-Choose from:
+Click on the step and select a click type you want the browser test to perform at execution time:
 
-* Primary click (corresponds to a left click)
+* Primary click corresponding to a left click
 * Double click
-* Contextual click (corresponds to a right click)
+* Contextual click corresponding to a right click
 
 ### Type text
 
-[Datadog browser test recorder extension][3] automatically records text input in any fields of your website (such as forms or text areas):
+Datadog records steps you perform on your application, such as selecting an option from a `select` dropdown menu, and a recap appears as a step on the left corner.
```

Comment:
explaining that what we show on the left corner is  a recap, why not 

Wouldn't it be "left panel" ?

Reply:
```suggestion
Datadog records steps you perform on your application, such as selecting an option from a `select` dropdown menu, and a recap appears as a step.
```